spec/schema.json

```diff
@@ -33,5 +51,131 @@
     }
   },
   "definitions": {
+    "OverseasEntitySubmission": {
+      "type": "object",
+      "properties": {
+        "presenter": {
+          "$ref": "#/definitions/Presenter"
+        },
+        "entity": {
+          "$ref": "#/definitions/Entity"
+        }
+      },
+      "required": [
+        "presenter",
+        "entity"
+      ]
+    },
+    "Presenter": {
+      "type": "object",
+      "properties": {
+        "fullName": {
+          "type": "string"
+        },
+        "phoneNumber": {
+          "type": "string"
+        },
+        "role": {
+          "type": "string",
+          "enum": [
+            "Administrator",
+            "Agent",
+            "Solicitor",
+            "Beneficial Owner",
+            "Other"
+          ]
+        },
+        "roleTitle": {
+          "type": "string"
+        },
+        "antiMoneyLaunderingRegistrationNumber": {
+          "type": "string"
+        }
+      },
+      "required": [
+        "fullName",
+        "phoneNumber",
+        "role",
+        "roleTitle"
+      ]
+    },
+    "Entity": {
+      "type": "object",
+      "properties": {
+        "entityName": {
+          "type": "string"
+        },
+        "incorporationCountry": {
+          "type": "string"
+        },
+        "principalAddress": {
+          "$ref": "#/definitions/Address"
+        },
+        "serviceAddress": {
+          "$ref": "#/definitions/Address"
+        },
+        "isAddressSameAsPrincipalAddress": {
+          "type": "boolean"
+        },
+        "email": {
+          "type": "string"
+        },
+        "legalForm": {
+          "type": "string"
+        },
+        "lawGoverned": {
+          "type": "string"
+        },
+        "publicRegisterEntityRegisteredOn": {
+          "type": "string"
+        },
+        "registrationNumber": {
+          "type": "string"
+        }
+      },
+      "required": [
+        "entityName",
+        "incorporationCountry",
+        "principalAddress",
+        "isAddressSameAsPrincipalAddress",
+        "email",
+        "legalForm",
+        "lawGoverned",
+        "publicRegisterEntityRegisteredOn",
+        "registrationNumber"
+      ]
+    },
+    "Address": {
+      "type": "object",
+      "properties": {
+        "propertyNameNumber": {
+          "type": "string"
+        },
+        "addressLine1": {
```

Comment:
Just line 1 and line 2 here - fields are in an 'address' object and you haven't prefixed the other fields with 'address'.